client/index.jsx

```diff
@@ -29,19 +29,20 @@ class App extends React.Component{
 
   componentDidMount(){
     this.getData('')
-    setTimeout(this.getData('Also'), 500);
+    setTimeout( this.getData( 'Also' ), 500);
   }
 
   getData(arg){
     /*
      window.location
     `${window.location.hostname}:3000/products/similar/`
     */
-
-    var url = `http://127.0.0.1:3000/products/similar/`
+ 
+    var url = `//${window.location.hostname}:3000/products/similar/${window.location.pathname.split('/')[1]}`
     if(arg!== ''){
-      url = `http://127.0.0.1:3000/products/alsolike/`;
+      url = `//${window.location.hostname}:3000/products/alsolike/${window.location.pathname.split('/')[1]}`;
```

Comment:
Have you tried this with the proxy? seems like it will try to route the api request through the proxy which could cause problems.

Reply:
just getting the current url , but it's work on my proxy server